### PR TITLE
cc Fix bad media condition in picture[sizes] test

### PR DESCRIPTION
**Pull request by sideshowbarker**
See https://github.com/w3c/css-validator/issues/266#issuecomment-561786647

### CODE REVIEW

File: conformance-checkers/tools/picture.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os
 ccdir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 template = """<!DOCTYPE html>
+<!-- DO NOT EDIT! This test has been generated by tools/picture.py. -->
```
Comment (zcorpan) [inline conformance-checkers/tools/picture.py]: Full root-relative path is probably more helpful for locating the file

```suggestion
<!-- DO NOT EDIT! This test has been generated by /conformance-checkers/tools/picture.py. -->
```

**Review by zcorpan — CHANGES_REQUESTED** [review]

File: conformance-checkers/tools/picture.py (hunk 1 above)
Comment (sideshowbarker) [reply conformance-checkers/tools/picture.py]: > Full root-relative path is probably more helpful for locating the file

Yep — made it so

**Review by zcorpan — APPROVED** [review]
LGTM, but there are failing checks. Maybe the checks need to bail early for changes in `conformance-checkers`?

**Comment by zcorpan** [comment]
Or maybe Taskcluster got confused by several force pushes?

> CRITICAL: task_head points at 2e4040fa890b3305de305fb0c0963e91d6b0d5d3, expected 83470bcb2cde679ad9d7361f6f086c4d642d12a0. This may be because the branch was updated

https://community-tc.services.mozilla.com/tasks/Ho-FhFDjQHuBxCWXbzCj9g/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FHo-FhFDjQHuBxCWXbzCj9g%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log

**Pull request merged**